web/pandas/pdeps/0005-no-default-index-mode.md

```diff
@@ -0,0 +1,189 @@
+# PDEP-5: No-default-index mode
+
+- Created: 14 November 2022
+- Status: Draft
+- Discussion: [#49693](https://github.com/pandas-dev/pandas/pull/49693)
+- Author: [Marco Gorelli](https://github.com/MarcoGorelli)
+- Revision: 1
+
+## Abstract
+
+The suggestion is to add a `mode.no_default_index` option which, if enabled,
+would ensure:
+- if a ``DataFrame`` / ``Series`` is created, then by default it won't have an ``Index``;
+- nobody will get an ``Index`` unless they ask for one - this would affect the default behaviour of ``groupby``, ``value_counts``, ``pivot_table``, and more.
```

Comment:
I think instead of saying "nobody will get an ``Index`` unless they ask for one", it would be better to say "all methods that create a default ``Index`` would no longer create one"